Makefile

```diff
@@ -13,7 +13,7 @@ clean:
 
 lint: SHELL:=/bin/bash
 lint:
-	golangci-lint run -c <(curl -s https://raw.githubusercontent.com/gruntwork-io/terragrunt/main/.golangci.yml) ./...
+	golangci-lint run -v ./...
```

Comment:
It's nice to have this in CI, but it's all good.

Reply:
It fails on lint v2, we can automate downloading in another step